forum/notifications/tests.py

```diff
@@ -1,52 +1,76 @@
 from django.test import TestCase
-from django.contrib.auth.models import User
-from .models import NotificationType, NotificationPreference
+from users.models import CustomUser
+from .models import (
+    NotificationType,
+    NotificationCategory,
+    NotificationPreference
+)
 from .serializers import (
     NotificationTypeSerializer,
+    NotificationCategorySerializer,
     NotificationPreferenceSerializer,
     NotificationPreferenceUpdateSerializer
 )
 
 
 class NotificationModelTests(TestCase):
     def setUp(self):
-        self.user = User.objects.create_user(username='testuser', password='password')
+        self.user = CustomUser.objects.create_user(email='test@user.com', password='password')
         self.notification_type = NotificationType.objects.create(
             name="email", description="Email notifications"
         )
+        self.notification_category = NotificationCategory.objects.create(
+            name="Message", description="You have message from some user"
+        )
         self.notification_preference, created = NotificationPreference.objects.get_or_create(user=self.user)
         self.notification_preference.allowed_notification_types.add(self.notification_type)
+        self.notification_preference.allowed_notification_categories.add(self.notification_category)
 
     def test_notification_type_creation(self):
         self.assertEqual(self.notification_type.name, "email")
+        self.assertEqual(self.notification_category.name, "Message")
         self.assertEqual(self.notification_type.description, "Email notifications")
+        self.assertEqual(self.notification_category.description, "You have message from some user")
```

Comment:
The method name suggests it tests the creation of `NotificationType`, but it also tests `NotificationCategory`.
Split this into two separate tests for clarity.

Reply:
Done ✅